plugins/woocommerce/tests/php/includes/class-wc-ajax-test.php

```
@@ -130,6 +130,136 @@ protected function skip_on_php_8_1() {
+	 * @testdox Should save term color only when adding visual attribute terms via AJAX.
+	 */
+	public function test_add_new_attribute_saves_color_only_for_visual_attributes(): void {
+		$original_theme      = wp_get_theme()->get_stylesheet();
+		$visual_attribute_id = null;
+		$text_attribute_id   = null;
+		$visual_taxonomy     = null;
+		$text_taxonomy       = null;
+		$visual_term_id      = 0;
+		$text_term_id        = 0;
+		$suffix              = (string) wp_rand( 1000, 9999 );
+		$administrator_role  = get_role( 'administrator' );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$administrator_role'. (undefined)

(UnusedLocalVariable)